examples/generate_certs.sh

```diff
@@ -0,0 +1,7 @@
+
+# use mkcert to generate certs for localhost
+mkcert localhost
```

Comment:
This is unrelated to our work, just makes it easier for people to get certs that are trusted by chrome

Reply:
@ten3roberts this script will help to generate certs that Google Chrome trusts, https://github.com/FiloSottile/mkcert If you are on MacOS you can `brew install mkcert`